addons/html_builder/static/src/core/building_blocks/builder_many2one.js

```diff
@@ -102,10 +105,31 @@ export class BuilderMany2One extends Component {
         return Promise.all(proms);
     }
     select(newSelected) {
+        this.preventNextPreview = false;
         this.callOperation(this.applyOperation.commit, {
             userInputValue: newSelected && JSON.stringify(newSelected),
         });
     }
+    preview(newSelected) {
+        if (this.preventNextPreview) {
+            return;
+        }
+        this.preventNextPreview = true;
```

Comment:
why do you need this preventNextPreview ? It s not done by default with callOperation ?

Reply:
I think the preventNextPreview stuff is duplicated in useClickableBuilderComponent/useInputBuilderComponent/useColorPickerBuilderComponent
We should maybe create hook to handle this stuff ?